test/test_torch.py

```diff
@@ -5609,6 +5609,42 @@ def test_copy_all_dtypes_and_devices(self, device):
             # not the data
             self.assertEqual(x, y)
 
+    @onlyNativeDeviceTypes
+    def test_copy_math_view(self, device):
+        for dst_dtype, src_dtype in [
+                (torch.float32, torch.float32),
+                (torch.float64, torch.float32),
+                (torch.int64, torch.int32),
```

Comment:
there's no way to have int tensors with neg bit no?

Reply:
Yes this isn't in the public API but since it is possible with the private API I don't think there's any harm in testing it.